conans/client/profile_loader.py

```diff
@@ -158,9 +158,11 @@ def _load_profile(self, profile_name, cwd):
 
         # All profiles will be now rendered with jinja2 as first pass
         base_path = os.path.dirname(profile_path)
+        file_path = os.path.basename(profile_path)
```

Comment:
``profile_name`` is already an argument to the function, maybe it can be used directly instead of needing to call ``os.path.basename``?

Reply:
The argument `profile_name` contains the full path in case passing the absolute path. I preferred taking from `profile_path`, because it's parsed previously by `get_profile_path`